scripts/pre_commit_linter.py

```diff
@@ -117,12 +117,12 @@
 BAD_PATTERNS = {
```

Comment:
In this file, please also make changes to disable the linting checks on CircleCI (though they should still run locally).